src/facilities/zlc/views/pages/Equipment.vue

```diff
@@ -0,0 +1,266 @@
+<!--
+  Adapted from https://github.com/Nikolai558/Equipment-Suffix-Generator under MIT license
+
+  NOTE: the unescaped HTML that's being shown here is part of a static
+  set of strings. No user input is being displayed in this manner.
+-->
+<!-- eslint-disable vue/no-v-html -->
```

Comment:
This is a no-no... please do not do this. Please use a dynamic component for this if you need the ability to render html within a component from a variable. There is a reason this is configured as an "error" level rule in eslint.

Reply:
It's "suggestion" level, not "problem" level.